node_modules/oae-content/tests/test-guests.js

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var assert = require('assert');
+
+var ContentTestUtil = require('oae-content/lib/test/util');
+var EmailAPI = require('oae-email');
+var PrincipalsTestUtil = require('oae-principals/lib/test/util');
+var RestAPI = require('oae-rest');
+var TestsUtil = require('oae-tests');
+
+describe('Content guests', function() {
+
+    // Rest contexts that can be used to make requests as different types of users
+    var camAdminRestContext = null;
+
+    /**
+     * Function that will fill up the rest contexts
+     */
+    before(function(callback) {
+        // Fill up the request contexts
+        camAdminRestContext = TestsUtil.createTenantAdminRestContext(global.oaeTests.tenants.cam.host);
+        return callback();
+    });
+
+    /**
+     * Test that verifies that content can be shared with a guest
+     */
+    it('verify a piece of content can be shared with a guest', function(callback) {
+        // Create some content and share it with a guest
+        ContentTestUtil.createContentAndInviteGuest(camAdminRestContext, function(simong, link, email, token) {
+
+            // Sign in as the guest
+            PrincipalsTestUtil.signIntoGuestAccountWithToken(token, function(loggedinGuest) {
+
+                // Verify that the link is in the guest's library
+                RestAPI.Content.getLibrary(loggedinGuest.restContext, loggedinGuest.user.id, null, null, function(err, data) {
+                    assert.ok(!err);
+                    assert.strictEqual(data.results.length, 1);
+                    assert.strictEqual(data.results[0].id, link.id);
+                    return callback();
+                });
+            });
+        });
+    });
+
+    /**
+     * Test that verifies that a piece of content can be shared guests when creating it
+     */
+    it('verify a piece of content can be shared with guests when creating it', function(callback) {
```

Comment:
None, I refactored the second one into the first one but forgot to remove the second one :f